dv_utils/client.py

```diff
@@ -59,7 +113,7 @@ def write_results(self, user_id: str, filename: str, content: str):
             raise Exception("Unsupported result file name: " + filename)
```

Comment:
This was not part of the PR, but I saw this and don't think we want Exceptions to be thrown in a cage but rather catch them and log

Reply:
✅